SPECS/php/php.spec

```diff
@@ -1,7 +1,6 @@
 %{!?_httpd_confdir:    %{expand: %%global _httpd_confdir %%{_sysconfdir}/httpd/conf.d}}
 %{!?_httpd_modconfdir: %{expand: %%global _httpd_modconfdir %%{_sysconfdir}/httpd/conf.modules.d}}
 %{!?_httpd_moddir:     %{expand: %%global _httpd_moddir %%{_libdir}/httpd/modules}}
-
 # Fedora spec file for php
 #
 # License: MIT
```

Comment:
Honestly, I think we can remove this header- we attribute and link to the license text in both the changelog and the license map file already.